JpaTransactionManager test throws NCDFE for ResourceTransactionManager

This seems to manifest itself when the tests are run in a particular order, thus it's not simple to replicate.
```
Caused by: java.lang.NoClassDefFoundError: org/springframework/transaction/support/ResourceTransactionManager
```

--- a/itests/standalone/basic/src/main/java/org/wildfly/camel/test/jpa/JpaTransactionManagerIntegrationTest.java
+++ b/itests/standalone/basic/src/main/java/org/wildfly/camel/test/jpa/JpaTransactionManagerIntegrationTest.java
@@ -31,13 +31,11 @@ import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.wildfly.camel.test.jpa.subA.Account;
 import org.wildfly.extension.camel.CamelContextRegistry;
 
-@Ignore("[wfc #872] JpaTransactionManager test throws NCDFE for ResourceTransactionManager")
 @RunWith(Arquillian.class)
 public class JpaTransactionManagerIntegrationTest {
 

--- a/modules/etc/smartics/skipped-modules.xml
+++ b/modules/etc/smartics/skipped-modules.xml
@@ -97,6 +97,10 @@
         <include artifact="org.apache.felix:(.*)" />
     </module>
     
+    <module name="org.apache.openjpa" skip="true">
+        <include artifact="org.apache.openjpa:(.*)" />
+    </module>
+    
     <module name="org.apache.servicemix.bundles" skip="true">
         <include artifact="org.apache.servicemix.bundles:(.*)" />
     </module>

--- a/modules/etc/smartics/spring-modules.xml
+++ b/modules/etc/smartics/spring-modules.xml
@@ -82,6 +82,9 @@
     
     <module name="org.springframework.orm">
         <include artifact=":spring-orm" />
+        <apply-to-dependencies skip="true">
+            <include module="org.apache.openjpa" />
+        </apply-to-dependencies>
         <dependencies>
             <module name="javax.api" />
             <module name="javax.persistence.api" />

